aws/sqs/sqs.go

```diff
@@ -157,10 +169,11 @@ func (s *SQS) receiveMessage(ctx context.Context, input *sqs.ReceiveMessageInput
 // which can be used to safely stop the system
 func (s *SQS) ReceiveWithContext(ctx context.Context, queueURL string, visibilityTimeout int32) (Raw, error) {
 	input := sqs.ReceiveMessageInput{
-		QueueUrl:            aws.String(queueURL),
-		MaxNumberOfMessages: 1,
-		VisibilityTimeout:   visibilityTimeout,
-		WaitTimeSeconds:     20,
+		QueueUrl:              aws.String(queueURL),
+		MaxNumberOfMessages:   1,
+		VisibilityTimeout:     visibilityTimeout,
+		WaitTimeSeconds:       20,
+		MessageAttributeNames: []string{"All"},
```

Comment:
If everything works as usual without this `MessageAttributeNames` then I'd suggest we don't add it.